spyder/plugins/variableexplorer/widgets/tests/test_namespacebrowser.py

```diff
@@ -15,9 +15,12 @@
 
 # Third party imports
 import pytest
+from qtpy.QtCore import Qt, QPoint
 
 # Local imports
 from spyder.plugins.variableexplorer.widgets.namespacebrowser import NamespaceBrowser
+from spyder.plugins.variableexplorer.widgets.tests.test_collectioneditor \
+    import data_table
```

Comment:
Please change this to

    from spyder.plugins.variableexplorer.widgets.tests.test_collectioneditor import data_table